types/rewards.yaml

```
@@ -0,0 +1,71 @@
+AttestationsRewards:
+  description: "Rewards info for attestations"
+  type: array
+  items:
+    $ref: "./rewards.yaml#/AttestationRewards"
+
+AttestationRewards:
+  type: object
+  description: "Rewards info for a single attestation"
+  required: ["validator_index", "head", "target", "source"]
+  properties:
+    validator_index:
+      allOf:
+        - $ref: "./primitive.yaml#/Uint64"
+        - example: 0
+        - description: "one entry for every validator based on their attestations in the epoch"
+    head:
+      allOf:
+        - $ref: "./primitive.yaml#/Int64"
+        - example: 2000
+        - description: "attester's reward for head vote in gwei"
+    target:
+      allOf:
+        - $ref: "./primitive.yaml#/Int64"
+        - example: 2000
+        - description: "attester's reward for target vote in gwei"
+    source:
+      allOf:
+        - $ref: "./primitive.yaml#/Int64"
+        - example: 4000
+        - description: "attester's reward for source vote in gwei"
+    inclusion_delay:
+      allOf:
+        - $ref: "./primitive.yaml#/Uint64"
+        - example: 2000
+        - description: "attester's inclusion_delay reward in gwei"
```

```suggestion
        - description: "attester's inclusion_delay reward in gwei (phase0 only)"
```

I think it's still good to indicate that this is optional, else consumers of the API might wonder why it's missing